unit2-csharp/lesson07.rst

```diff
@@ -18,5 +18,7 @@ Objectives
 - Explain what a default method is and how they may be used with interfaces
 - Use the appropriate interface for collection types depending on the behavior they must exhibit
 - Describe the differences between abstract classes and interfaces
-- Use the ``Comparator.compare()`` method
+- Use some C# provided interfaces: ``IComparer<T>``, ``IEnumerable<T>``, ``IList<T>``, ``Dictionary<TKey, TValue>``
```

Comment:
Is this supposed to be ``IDictionary`` or ``Dictionary``?

Reply:
yes IDictionary ty!